src/extension.ts

```diff
@@ -63,6 +63,7 @@ export async function activate(context: ExtensionContext) {
         }
       });
     });
+  commands.executeCommand("setContext", "metals:enabled", true);
```

Comment:
😍 this is a beautiful trick I didn't know about!

Reply:
This is awesome! I've been frustrated by not having "Metals: Import build" when editing non-scala files.